### PR TITLE
Hotfix: update event tracking

### PR DESCRIPTION
**Pull request by garethfuller**
# Description

Some events had been dropped in previous refactoring and some were named incorrectly, this PR fixes those things and adds a couple more events.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [x] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

- [ ] Test app functions as normal.

## Visual context

n/a

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/trade/useSor.ts [hunk 1]
```diff
@@ -496,7 +496,7 @@ export default function useSor({
       onTxConfirmed: () => {
         trading.value = false;
         latestTxHash.value = tx.hash;
-        trackGoal(Goals.Swapped);
+        trackGoal(Goals.Swapped, Number(tokenInAmountInput.value) || 0);
```
Comment (timjrobinson) [inline src/composables/trade/useSor.ts:499]: Is there a way to add the token symbol or USD value here? as the token amount by itself isn't a very useful metric .

**Review by timjrobinson — APPROVED** [review]
One comment, otherwise LGTM

**Review by Matthews3301 — APPROVED** [review]
Looks okay, I'm not 100% sure about the purpose of specific tracking events, however, would it not be better to track pages being opened via `beforeMount` on each page rather than link clicks? Some links don't have the tacking behaviour, e.g. in the Footer.

**Comment by garethfuller** [comment]
> Looks okay, I'm not 100% sure about the purpose of specific tracking events, however, would it not be better to track pages being opened via `beforeMount` on each page rather than link clicks? Some links don't have the tacking behaviour, e.g. in the Footer.

@Matthews3301 Yeah good point, but individual page views are tracked, the click events are more just about understanding what users are using to navigate the site. They probably aren't that useful but the more data the better I guess.

**Pull request merged**